.github/actions/setup-environment/action.yml

```diff
@@ -8,6 +8,9 @@ inputs:
   tpm_enabled:
     required: true
     type: bool
+  eve_tag:
```

Comment:
As pointed out we need to propagate this to `run-eden-test` and also I'd keep in mind that we want to run this workflow in eden as well (to be sure that changes in eden don't affect existing tests) and right now we don't set eve tag to run this tests, so maybe it could be a good idea to make it optional and setup eve version only when eve_tag is set